integration_tests/suite/helpers/constants.py

```diff
@@ -1,13 +1,21 @@
-# Copyright 2016-2019 The Wazo Authors  (see the AUTHORS file)
+# Copyright 2016-2022 The Wazo Authors  (see the AUTHORS file)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 import os.path
 
+from uuid import UUID
+
 ASSET_ROOT = os.path.join(os.path.dirname(__file__), '..', '..', 'assets')
 
 INVALID_TOKEN_ID = 'invalid-token'
 UNAUTHORIZED_TOKEN_ID = 'invalid-acl-token'
 
+TOKEN_UUID = UUID('00000000-0000-0000-0000-0123456789ab')
+USER_UUID = UUID('00000000-0000-0000-0000-00000000beef')
+MASTER_TENANT_UUID = UUID('00000000-0000-0000-0000-000000000001')
+TENANT1_UUID = UUID('00000000-0000-0000-0000-000000000002')
+TENANT2_UUID = UUID('00000000-0000-0000-0000-000000000003')
```

Comment:
Use uuid4 format. It can avoid error when parsing by service (it's not the case here)
```suggestion
TOKEN_UUID = UUID('00000000-0000-4000-a000-0123456789ab')
USER_UUID = UUID('00000000-0000-4000-a000-00000000beef')
MASTER_TENANT_UUID = UUID('00000000-0000-4000-a000-000000000001')
TENANT1_UUID = UUID('00000000-0000-4000-a000-000000000002')
TENANT2_UUID = UUID('00000000-0000-4000-a000-000000000003')
```